ansible/bootstrap.yml

```diff
@@ -21,7 +21,7 @@
         packages:
           - mpd
           - mpc
-          - openjdk-8-jdk-headless
```

Comment:
to be discussed -> ZERO compatibility due to non existing FPU
 * JAVA 8 lock
 * alternative JVM